scaleway/terraform/cluster.tf

```diff
@@ -0,0 +1,22 @@
+resource "scaleway_k8s_cluster" "this" {
```

Comment:
Please use proper resource names here and in other places instead of `this` and `current`